### ISSUE TITLE
Sync already created branches

### ISSUE THREAD

**Issue by alve89** [atom/github#2416]
I'm quite sure I'm not the first one who brings that request, but only be able to create new branches locally without checking if there are new ones (ore locally still existing but on the remote already deleted ones) and "pulling" them to the local machine, is only half-helpful.
I love to use Atom on two clients but without GitHub Desktop I can't use new branches - or am I just misunderstanding something?

So, long story short: Please sync branches between Atom Editor and GitHub. 😊

**Comment by DarkGuy10** [comment]
exactly, i want that too
